ETL/dynamodb-etl.sh

```diff
@@ -582,16 +638,30 @@ for worker in $(seq 0 $((WORKERS - 1))); do
 	PIPE_NAME[$worker]="$PIPE"
 	worker "${segment}" > "$PIPE" &
 	: {FD}< "${PIPE}"
+	WORKER_PID["${worker}"]=$!
 	PIPE_FD["${worker}"]="${FD}"
 done
 
+declare -a MAIN_PID
+
 if [[ -n $STDOUT ]]; then
 	main_stdout
 else
-	main_pipe
+	for index in "${!PIPE_NAME[@]}"; do
+		main_pipe "$index" &
+		MAIN_PID["${index}"]=$!
+	done
+fi
+
+declare WAIT_FOR
```

Comment:
do we still need this `WAIT_FOR` var even though online 273 we already have a `declare -g WAIT_FOR`?